### PR TITLE
Calendar click on description

### LINKED ISSUE #59: Create a function to launch an appointment's room when clicking on it in the calendar.

### PR DESCRIPTION
**Pull request by AttiaYoussef**
Resolves #59 
Can now click on an appointment's description to launch its room.

### CODE REVIEW

File: app/src/main/res/layout/activity_calendar2.xml [hunk 1]
```diff
@@ -46,6 +46,31 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_begin="20dp"
+        app:layout_constraintGuide_percent="0.20" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline7"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
```
Comment (MrZaiko) [inline app/src/main/res/layout/activity_calendar2.xml]: Ids not correct

File: app/src/main/res/layout/activity_calendar_entry.xml [hunk 2]
```diff
@@ -30,12 +35,28 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_begin="20dp"
+        app:layout_constraintGuide_percent="0.0" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline4"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
```
Comment (MrZaiko) [inline app/src/main/res/layout/activity_calendar_entry.xml]: Same here

**Review by MrZaiko — CHANGES_REQUESTED** [review]

**Review by MrZaiko — APPROVED** [review]
Perfect!

**Review by MRandl — APPROVED** [review]
all good :)

**Comment by MRandl** [comment]
how dare you bring -0.2% code coverage, this is unacceptable

**Pull request merged**